source/guides/dashboard/introduction.md

```diff
@@ -28,6 +28,13 @@ From the Dashboard you can:
 
 {% imgTag /img/dashboard/dashboard-runs-list.png "Dashboard Screenshot" %}
 
+## Manage runs
+
+From the Dashboard you can:
+
+- Cancel runs currently in progress
+- Archive runs in a cancelled or errored state
```

Comment:
Same - link to section using a url tag. 

If you're running `npm start` locally to see docs, it will warn in the terminal if an invalid url is written in the url tag, so it's good to have this running when making changes.